File: ios/CrnlNitro.swift
class CrnlNitro: HybridCrnlNitroSpec {
    public var hybridContext = margelo.nitro.HybridContext()
    public var memorySize: Int {
        getSizeOf(self)
    }

    public func multiply(a: Double, b: Double) throws -> Double {
        return a * b
    }
}

File: android/src/main/java/com/margelo/nitro/crnlnitro/CrnlNitro.kt
package com.margelo.nitro.crnlnitro

class CrnlNitro : HybridCrnlNitroSpec() {
  override val memorySize: Long
    get() = 0L

  override fun multiply(a: Double, b: Double): Double {
    return a * b
  }
}
